dotnet/src/Microsoft.AutoGen/Agents/Services/HostBuilderExtensions.cs

```diff
@@ -28,12 +28,12 @@ public static IHostApplicationBuilder AddAgent<
     public static IHostApplicationBuilder AddAgent(this IHostApplicationBuilder builder, string typeName, Type agentType)
     {
         builder.Services.AddKeyedSingleton("AgentTypes", (sp, key) => Tuple.Create(typeName, agentType));
-
         return builder;
     }
 
-    public static IHostApplicationBuilder AddAgentWorker(this IHostApplicationBuilder builder, string agentServiceAddress = _defaultAgentServiceAddress, bool local = false)
+    public static IHostApplicationBuilder AddAgentWorker(this IHostApplicationBuilder builder, string? agentServiceAddress = null, bool local = false)
```

Comment:
maybe still make agentServiceAddress  required and assgin default value to it?

Reply:
this lets use use the environment var if its not set